### PR TITLE
pass services instead of labels to ServicesCli#kill

### PR DESCRIPTION
**Pull request by haacs**
fixes brew trying to call kill with a bunch of strings when orphaned services are found

### CODE REVIEW

File: lib/service/services_cli.rb [hunk 1]
```diff
@@ -37,14 +37,18 @@ def check(targets)
     # Kill services that don't have a service file
     def kill_orphaned_services
       cleaned = []
+      cleaned_svcs = []
```
Comment (MikeMcQuaid) [inline lib/service/services_cli.rb:40]:
```suggestion
      cleaned_lables = []
      cleaned_services = []
```

File: lib/service/services_cli.rb [hunk 2]
```diff
@@ -37,14 +37,18 @@ def check(targets)
...
       running.each do |label|
         if (svc = FormulaWrapper.from(label))
```
Comment (MikeMcQuaid) [inline lib/service/services_cli.rb:42]:
```suggestion
        if (service = FormulaWrapper.from(label))
```

**Review by MikeMcQuaid — COMMENTED** [review]
Nice work so far! A few naming nits.

File: lib/service/services_cli.rb (hunk 1 above)
Comment (SMillerDev) [reply lib/service/services_cli.rb:40]: Shouldn't it be `labels`?

Comment (MikeMcQuaid) [reply lib/service/services_cli.rb:40]: Yeh, sorry, fixed.

**Comment by haacs** [comment]
thanks for the feedback :)
should i squash these commits before you merge or do you do a squash merge?

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Looks good to me! Will let @SMillerDev merge when he's ✅ as I'm on holiday so fairly unresponsive if this needs reverted.

**Review by SMillerDev — APPROVED** [review]

**Pull request merged**